app/form/v1_0/fields/drug-use/add-drugs.ts

```diff
@@ -12,27 +30,27 @@ const selectMisusedDrugs: FormWizard.Field = {
   multiple: true,
   validate: [{ type: ValidationType.Required, message: 'Select which drugs they have used' }],
   options: [
-    { text: 'Amphetamines (including speed, methamphetamine)', value: 'AMPHETAMINES', kind: 'option' },
-    { text: 'Benzodiazepines (including diazepam, xanex)', value: 'BENZODIAZEPINES', kind: 'option' },
-    { text: 'Cannabis', value: 'CANNABIS', kind: 'option' },
-    { text: 'Cocaine', value: 'COCAINE', kind: 'option' },
-    { text: 'Crack cocaine', value: 'CRACK', kind: 'option' },
-    { text: 'Ecstasy (MDMA)', value: 'ECSTASY', kind: 'option' },
-    { text: 'Hallucinogenics (including ketamine)', value: 'HALLUCINOGENICS', kind: 'option' },
-    { text: 'Heroin', value: 'HEROIN', kind: 'option' },
-    { text: 'Methadone (not prescribed)', value: 'METHADONE_NOT_PRESCRIBED', kind: 'option' },
-    { text: 'Prescribed drugs', value: 'MISUSED_PRESCRIBED_DRUGS', kind: 'option' },
-    { text: 'Other opiates', value: 'OTHER_OPIATES', kind: 'option' },
-    { text: 'Solvents (including gases and glues)', value: 'SOLVENTS', kind: 'option' },
-    { text: 'Steroids', value: 'STEROIDS', kind: 'option' },
-    { text: 'Synthetic cannabinoids (spice)', value: 'SPICE', kind: 'option' },
-    { text: 'Other', value: 'OTHER_DRUG', kind: 'option' },
+    { text: drugsList.AMPHETAMINES.text, value: drugsList.AMPHETAMINES.value, kind: 'option' },
+    { text: drugsList.BENZODIAZEPINES.text, value: drugsList.BENZODIAZEPINES.value, kind: 'option' },
+    { text: drugsList.CANNABIS.text, value: drugsList.CANNABIS.value, kind: 'option' },
+    { text: drugsList.COCAINE.text, value: drugsList.COCAINE.value, kind: 'option' },
+    { text: drugsList.CRACK.text, value: drugsList.CRACK.value, kind: 'option' },
+    { text: drugsList.ECSTASY.text, value: drugsList.ECSTASY.value, kind: 'option' },
+    { text: drugsList.HALLUCINOGENICS.text, value: drugsList.HALLUCINOGENICS.value, kind: 'option' },
+    { text: drugsList.HEROIN.text, value: drugsList.HEROIN.value, kind: 'option' },
+    { text: drugsList.METHADONE_NOT_PRESCRIBED.text, value: drugsList.METHADONE_NOT_PRESCRIBED.value, kind: 'option' },
+    { text: drugsList.MISUSED_PRESCRIBED_DRUGS.text, value: drugsList.MISUSED_PRESCRIBED_DRUGS.value, kind: 'option' },
+    { text: drugsList.OTHER_OPIATES.text, value: drugsList.OTHER_OPIATES.value, kind: 'option' },
+    { text: drugsList.SOLVENTS.text, value: drugsList.SOLVENTS.value, kind: 'option' },
+    { text: drugsList.STEROIDS.text, value: drugsList.STEROIDS.value, kind: 'option' },
+    { text: drugsList.SPICE.text, value: drugsList.SPICE.value, kind: 'option' },
+    { text: drugsList.OTHER_DRUG.text, value: drugsList.OTHER_DRUG.value, kind: 'option' },
   ],
   labelClasses: utils.getMediumLabelClassFor(FieldType.CheckBox),
```

Comment:
isn't this the field which should be setting `utils.visuallyHidden` ? Or am I misunderstanding this?